tests/src/system/basic/WskBasicTests.scala

```diff
@@ -87,48 +87,69 @@ class WskBasicTests
         stdout should include regex ("""(?i)whisk API version\s+v1""")
     }
 
+    it should "validate default property values" in {
+        val wskpropsBackup = TestUtils.backupWskProps()
```

Comment:
Sorry to keep repeating the same thing - but why do this vs 
`val env = Map("WSK_CONFIG_FILE" -> tmpwskprops.getAbsolutePath())` which is used in other tests that much around with saved properties?